app/services/brewery_search.rb

```diff
@@ -0,0 +1,29 @@
+class BrewerySearch
+  def self.find_brewery(name)
+    new(name).find_brewery(name)
+  end
+
+  def initialize(name)
+    @name = name
```

Comment:
Unused i-var.

Reply:
Fixed this with the refactoring, thanks!